classes/bridges/squid.js

```diff
@@ -79,7 +60,7 @@ class SquidBridge extends BaseBridge {
           quoteOnly: false,
         };
 
-        const { route } = await this.getRouteWithCache(routeParams);
+        const route = await this.getRoute(routeParams);
         this.feeCosts = route.estimate?.feeCosts?.[0]?.amountUsd;
         return this.feeCosts;
       } catch (error) {
```

Comment:
I mean this part, we should call `getFeeCosts()` here to make it more maintainable